components/stepped-form/index.tsx

```diff
@@ -184,6 +194,22 @@ const SteppedForm = () => {
     initForm();
   }, []);
 
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+    // array of objects containing id and value from input
+    const responses = []
+    // get id and values from input field
+    for (const response of event.target) {
+      responses.push({id: response.name, value: response.value})
```

Comment:
would a .map or .foreach or even a .reduce be better here than a loop, havent seen var loops in js in a while

Reply:
yes - although depends on use case. `for...of` is good for readability and only incrementally slower. had I more brain cells the ideal solution is a basic `.map`